telemetry/definitions/commonDefinitions.json

```
@@ -554,6 +554,11 @@
             "type": "string",
             "description": "Describes which parts of an application (that we know of) were synced to the cloud. \"Code\" resources follow the SAM spec: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-sync.html",
             "allowedValues": ["AllResources", "CodeOnly"]
+        },
+        {
+            "name": "startUrl",
```

- we have `credentialModification`, `credentialSourceId`, etc. Perhaps this should be named `credentialUrl` or `credentialStartUrl` to use a consistent prefix.
- can the start URL have PII?

```suggestion
            "name": "credentialStartUrl",
```